noxfile.py

```diff
@@ -108,7 +108,7 @@ def _run_integration_tests(session: Session, env: dict[str, str]) -> None:
     netrc = "machine 127.0.0.1 login cachi2-user password cachi2-pass"
     default_env = {"HERMETO_TEST_NETRC_CONTENT": os.getenv("HERMETO_TEST_NETRC_CONTENT", netrc)}
     default_env.update(env)
-    cmd = "pytest --log-cli-level=WARNING tests/integration"
+    cmd = "pytest --log-cli-level=WARNING -W ignore::DeprecationWarning tests/integration"
```

Comment:
This one sneaked in, but I do find it useful. If someone else finds it useful I'll re-home this change into a separate commit else I'll drop it in the next revision.

Reply:
Moved to a separate commit. Will drop in case there is any opposition to it.